canonical/assets.ubuntu.com
# WD-13749 endpoint for returning list of employees for author

## Done

- Added endpoint for the fetching list of employees
## QA
You would need `DIRECTORY_API_TOKEN` and be on VPN for the API to work. Please reach out for token.

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8017
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- This API requires an authToken, either directly add a token into the `token` table of the postgres database or use cli commands to add one.
- Once you have added the token, use GET request on url `http://0.0.0.0:8017/v1/get-users/<keyword to search for>?token=<your token>` .
- It should return a list of json objects with relevant match.

## Issue / Card

Fixes # [WD-13749](https://warthogs.atlassian.net/browse/WD-13749)



[WD-13749]: https://warthogs.atlassian.net/browse/WD-13749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
